CONTRIBUTING.md

```diff
@@ -626,9 +624,43 @@ To install DocFX on a Microsoft Windows machine:
 > [!TIP]
 > Alternative ways to install DocFX can be found on the [DocFX website](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool).
 
-#### Making a test build
+#### First-time setup
+
+Before you make your first test build, we recommend the first-time setup detailed below. If you do not do this, you will still be able to make a test build, but there may be many warnings in the terminal, and some pages may not be included in the build.
+
+1. Check if your fork is up to date with the latest changes from the main dataminer-docs repository (See [Make sure your fork is up to date](#make-sure-your-fork-is-up-to-date).
+
+1. Make sure you have Visual Studio 2017 or higher installed.
+
+1. Install the .NET framework SDK Developer Pack. To do so, go to <https://dotnet.microsoft.com/en-us/download/visual-studio-sdks> and choose the .NET Framework 4.8 Developer Pack.
+
+1. Download and install [Build Tools](https://aka.ms/vs/17/release/vs_BuildTools.exe).
```

Comment:
The Build tools are not required if you have Visual Studio installed. (In fact, DocFX states that is requires VS (so I assume using Build tools instead of VS would not work based on their documentation: https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html). It also states VS2019 (or above) is required while here VS2017 is mentioned.